src/containers/AdminCampaignEdit.jsx

```diff
@@ -362,43 +363,53 @@ export class AdminCampaignEditBase extends React.Component {
       if (newCampaign.hasOwnProperty("interactionSteps")) {
         newCampaign.interactionSteps = makeTree(newCampaign.interactionSteps);
       }
-
+      
       await this.props.mutations.editCampaign(
         this.props.campaignData.campaign.id,
         newCampaign
-      );
-    }
-  };
-
-  checkSectionSaved(section) {
-    // Tests section's keys of campaignFormValues against props.campaignData
-    // * Determines greyness of section button
-    // * Determine if section is marked done (in green) along with checkSectionCompleted()
-    // * Must be false for a section to save!!
-    // Only Contacts section implements checkSaved()
-    if (section.hasOwnProperty("checkSaved")) {
-      return section.checkSaved();
+        );
+      }
+    };
+    
+    checkSectionSaved(section) {
+      // Tests section's keys of campaignFormValues against props.campaignData
+      // * Determines greyness of section button
+      // * Determine if section is marked done (in green) along with checkSectionCompleted()
+      // * Must be false for a section to save!!
+      // Only Contacts section implements checkSaved()
+      if (section.hasOwnProperty("checkSaved")) {
+        return section.checkSaved();
+      }
+      const sectionState = {};
+      const sectionProps = {};
+      section.keys.forEach(key => {
+        sectionState[key] = this.state.campaignFormValues[key];
+        sectionProps[key] = this.props.campaignData.campaign[key];
+      });
+      if (JSON.stringify(sectionState) !== JSON.stringify(sectionProps)) {
+        return false;
+      }
+      // console.log("SectionState", sectionState)
+      // console.log("SectionProps", sectionProps)
```

Comment:
These console.logs were not as interesting as actually getting this.state or this.props